### DISCUSSION TITLE
Item above chest

### DISCUSSION THREAD

**Discussion by Skw972** [PotatoCraft-Studio/QuickShop-Reremake#202] [Ideas]
If you put a custom item that has custom texture set in optifine it looks really ugly. It's because item in preview is normal but item above chest is just choosen material. When you put stone named "&6Stone" and in texturepack you add texture to it it looks really ugly. Having two different items.

**Comment by sandtechnology** [comment]
So it's a bug report or what?
Could you give an screenshot for this issue?
